src/applications/vaos/appointment-list/components/RequestedStatusAlert.jsx

```diff
@@ -59,17 +59,20 @@ export default function RequestedStatusAlert({ appointment, facility }) {
             available appointment.
             <br />
             <div className=" vads-u-margin-top--1">
-              <Link to="/" onClick={handleViewClick}>
-                View your appointments
-              </Link>
+              <va-link
+                href="/"
```

Comment:
This link is going to the wrong path. "/" was enrolled in a special routes file. I don't know how it works but @vbahinwillit may have the background on it.

Reply:
> The details link is working perfectly but the links in the status alert is going to the wrong path.

@jenniemc I changed this to use the va-link component with the React bindings to handle the onclick event.